frontend/src/page/routine.rs

```diff
@@ -1071,42 +1071,39 @@ fn view_routine_part(
                                     ]
                                 ]
                             ],
-                            IF![
-                                show_tut =>
+                            div![
+                                C!["field"],
+                                C!["mb-0"],
+                                C!["mr-2"],
                                 div![
```

Comment:
Removing this condition makes it possible to set a target time for exercises, but this value will still not be shown in training sessions. I think it would make more sense to change this condition to `show_tut || exercise_id.is_none()`, so that the time field is only shown for rests if TUT is disabled.

Reply:
You're right - that makes much more sense. I didn't fully grasp how the rendering was performed here (again :wink:). Changed.